app/serializers/cape_serializer.rb

```diff
@@ -1,6 +1,7 @@
 class CapeSerializer < ActiveModel::Serializer
-  attributes :id, :name, :fullName, :gender, :affiliation, :intelligence, :strength, :speed, :url
+  attributes :id, :name, :fullName, :gender, :affiliation, :intelligence, :strength, :speed, :url, :user_id
```

Comment:
user_id will be blank in this case. Remove it, your belongs to will add the user

or make a custom method that just returns user_id